gradle.properties

```diff
@@ -18,22 +18,22 @@ kotlinJvmVersion = 1.8
 kotlinVersion = 1.3.72
 kotlinCoroutinesVersion = 1.3.8
 
-classGraphVersion = 4.8.87
+classGraphVersion = 4.8.90
 graphQLJavaVersion = 15.0
 jacksonVersion = 2.11.3
 kotlinPoetVersion = 1.6.0
 ktorVersion = 1.3.1
-reactorVersion = 3.3.10.RELEASE
-reactorExtensionsVersion = 1.0.2.RELEASE
-springBootVersion = 2.3.4.RELEASE
-springVersion = 5.2.9.RELEASE
+reactorVersion = 3.4.0
```

Comment:
spring boot `2.3.5` references reactor BOM version `Dysprosium-SR13` which in turn defines reactor version `3.3.11.RELEASE` and extensions as `1.0.2.RELEASE` -> should we keep those in sync?